# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/semantics/popovers/popover-top-layer-interactions.html.ini
# Commit: 46fd81530a8b
# Full Hash: 46fd81530a8b6d81b5520aff9451d49649f1420a
# Author: Ziran Sun <zsun@igalia.com>
# Date: 2023-05-23 09:47:12
# Regressor Bug: 1832837
# File Overlap Count: 1
# Description:
#   Bug 1832837 - Remove the fullscreen element ready check on fullscreen element descendant. r=edgar
#   
#   https://fullscreen.spec.whatwg.org/#fullscreen-element-ready-check
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D177913
# ==============================================================================

diff -r 069f4fb8fd87 -r 46fd81530a8b testing/web-platform/meta/html/semantics/popovers/popover-top-layer-interactions.html.ini
--- a/testing/web-platform/meta/html/semantics/popovers/popover-top-layer-interactions.html.ini	Mon May 22 11:53:20 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,15 +0,0 @@
-[popover-top-layer-interactions.html]
-  [A Fullscreen Element should *not* close a Fullscreen Element.]
-    expected: FAIL
-
-  [A Popover should *not* close a Modal Dialog.]
-    expected:
-      if not debug and (os == "linux"): [PASS, FAIL]
-      if not debug and (os == "android"): [PASS, FAIL]
-      if not debug and (os == "mac"): [PASS, FAIL]
-
-  [A Popover should *not* close a Fullscreen Element.]
-    expected:
-      if not debug and (os == "linux"): [PASS, FAIL]
-      if not debug and (os == "android"): [PASS, FAIL]
-      if not debug and (os == "mac"): [PASS, FAIL]
